### PR TITLE
Add parm -> param

### PR DESCRIPTION
**Pull request by lurch**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -19415,11 +19415,13 @@ paritition->partition
 parititions->partitions
 parituclar->particular
 parliment->parliament
+parm->param
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:19418]: Also suggest pram and Parma (like the ham).

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -19415,11 +19415,13 @@ paritition->partition
...
 parmaeter->parameter
 parmaeters->parameters
 parmaters->parameters
 parmeter->parameter
 parmeters->parameters
+parms->params
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:19424]: Suggest prams too.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (lurch) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (lurch) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**